README.md

```diff
@@ -84,6 +84,10 @@ that can't be shared publicly. More documentation and guides can be found on the
 
 Read our [Contributing Guide](CONTRIBUTING.md) to learn about reporting issues, contributing code, and more ways to contribute.
 
+## Security
+
+If you happen to find a security vulnerability, we would appreciate you letting us know at https://hackerone.com/automattic and allow us to respond before disclosing the issue publicly.
```

Comment:
Can we change "allow us" to "allowing us"? 

Then good to :shipit:!

Reply:
👍